src/TestInfrastructure/VsObjectModelMocks/ProjectMock.cs

```diff
@@ -294,9 +294,12 @@ int IVsProject.AddItem(uint itemidLoc, VSADDITEMOPERATION dwAddItemOperation, st
         int IVsBuildPropertyStorage.GetPropertyValue(string pszPropName, string pszConfigName, uint storage, out string pbstrPropValue)
         {
             pbstrPropValue = null;
-            return this.buildProperties.TryGetValue(pszPropName, out pbstrPropValue)
-                ? VSConstants.S_OK
-                : VSConstants.E_FAIL;
+            if (this.buildProperties.ContainsKey(pszPropName))
```

Comment:
if it doesn't contain it should we return E_FAIL or that unchecked value for which you added a const somewhere above?